packages/genesys-spark-components/src/components/stable/gux-calendar/gux-calendar.less

```diff
@@ -69,6 +69,7 @@
   .gux-content {
     display: flex;
     align-items: flex-start;
+    height: 190px;
```

Comment:
I think this height suits better for the `v3` version in `v4` the token is 234px but it does not look good in `v3` as the paddings and stuff are different.

Reply:
<img width="518" alt="Screenshot 2024-06-14 at 10 39 27" src="https://github.com/MyPureCloud/genesys-spark/assets/25255508/bed340c7-8115-4bd3-9361-fa867c3ee6db">

I think the single calendar is getting cut a little too short here. 196px seems like it might work better there.